src/Microsoft.DotNet.CodeAnalysis/PackageFiles/codeAnalysis.targets

```diff
@@ -7,4 +7,8 @@
     <AdditionalFiles Include="$(MSBuildProjectDirectory)\*.analyzerdata" />
     <AdditionalFiles Include="$(MSBuildProjectDirectory)\*.analyzerdata.$(Platform)" />
   </ItemGroup>
+
+  <PropertyGroup>
+  	<CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)Microsoft.DotNet.CodeAnalysis.ruleset</CodeAnalysisRuleSet>
```

Comment:
If we do, they can use the $(OverrideCodeAnalysisRuleSet) to specify which ruleset to use for that OS.